devpack/src/main/java/io/neow3j/devpack/framework/annotations/ManifestFeature.java

```diff
@@ -0,0 +1,15 @@
+package io.neow3j.devpack.framework.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Target;
+
+@Target(ElementType.TYPE)
+public @interface ManifestFeature {
+
+//    String name() default "name";
```

Comment:
Are those attributes not supported now, but they will, in the future? If yes, add TODO

Reply:
Forgot to remove the commented attributes. They shouldn't be here.